src/plugins/lua/replies.lua

```diff
@@ -81,7 +86,12 @@ local function replies_check(task)
         return rcpt.addr or ''
       end, rcpts)) then
         lua_util.debugm(N, task, 'reply to %s validated', in_reply_to)
-        return true
+
+        --storing only addr of rcpt
+        for i = 1, #rcpts, 1 do
```

Comment:
Why do we need a step here:
```suggestion
        for i = 1, #rcpts do
```